src/parsers/manifestjson.js

```diff
@@ -121,8 +124,17 @@ export default class ManifestJSONParser extends JSONParser {
         }
       }
     }
+
     if (support.firefox_android) {
-      const versionAddedAndroid = support.firefox_android.version_added;
+      // `@mdn/browser-compat-data` sometimes provides data with gaps, e.g., a
+      // feature was supported in Fennec (added in 56 and removed in 79) and
+      // then re-added in Fenix (added in 85) and this is expressed with an
+      // array of objects instead of a single object.
+      //
+      // This is the case of the `permissions.browsingData` on Android for
+      // instance.
```

Comment:
Do we also want to mention here that we explicitly decided to:
> only warn the developer if the minVersion required by the extension is not greater or equal of the first version where the feature was officially supported for the first time (and do not warn if the minVersion is in one of the few version gaps).

what do you think?